File: Service/MediCare/RMA.Service.MediCare.Contracts/Entities/Medical/PreAuthTreatmentBasket.cs
namespace RMA.Service.MediCare.Contracts.Entities.Medical
{
    public class PreAuthTreatmentBasket : Common.Entities.AuditDetails
    {
        public int PreAuthTreatmentBasketId { get; set; }
        public int PreAuthId { get; set; }
        public int TreatmentBasketId { get; set; }
        public bool? IsAuthorised { get; set; }
        public bool IsClinicalUpdate { get; set; }
        public short? UpdateSequenceNo { get; set; }
        public int? ClinicalUpdateId { get; set; }
    }
}

File: Web/RMA-Web-Shell/projects/medicare/src/app/preauth-manager/models/preauth-treatment-basket.ts
export class PreAuthTreatmentBasket
{
    createdBy: string;
    createdDate: Date;
    isAuthorised: boolean;
    isClinicalUpdate: boolean;
    modifiedBy: string;
    modifiedDate: Date;
    preAuthTreatmentBasketId: number;
    treatmentBasketId: number;
    description:string;
    icd10CodeId:number;
    preAuthId?: number;
    updateSequenceNo: number;
}